backend/dataall/api/Objects/ShareObject/resolvers.py

```diff
@@ -240,7 +240,7 @@ def resolve_user_role(context: Context, source: models.ShareObject, **kwargs):
                 or dataset.SamlAdminGroupName in context.groups
             )
         ):
-            return ShareObjectPermission.ApproversAndRequesters.value
+            return ShareObjectPermission.Approvers.value
```

Comment:
I think this logic is repeated in line 225-226. Maybe we can have a single case that returns Approvers

Reply:
It seems so. I will remove the check